frontend/src/index.pug

```diff
@@ -206,6 +206,9 @@ html
     vuetemplate#v_zoom
       include tabs/zoom.pug
 
+    vuetemplate#v_segment
+      include segment.pug
```

Comment:
Maybe `tabs/segment` will be better? I think @ongspxm is planning to extract more template into its own file. And this `segment` belongs to ` tab`.